.github/workflows/deploy.yml

```diff
@@ -98,3 +98,15 @@ jobs:
       - name: Check sourcemaps
         run: |
           [[ ${{ needs.sourcemaps.result }} =~ success|skipped ]]
+
+      - name: Notify Slack on failure in merge queue
+        uses: rtCamp/action-slack-notify@19c86e40d4bc3be986b18f1459d4549536887065
+        if: ${{ failure() && github.event_name == 'merge_group' }}
+        env:
+          SLACK_LINK_NAMES: true
+          SLACK_MESSAGE: "At least one deployment job failed for a Pull Request in the Merge Queue failed <@U0143NL4GMP> <@U027NPY8Y3X> <@U02NLJY0FGX>"
```

Comment:
There's no way to tag the correct person/group on a per-assignee or merge job creator basis? Wondering if this might get spammy (e.g. for overnight Renovate PRs that are NGMI).